### PR TITLE
Update README.md release instructions

### PR DESCRIPTION
**Pull request by kellyi**
## Overview

This PR adjusts the README's release section to note that we use git flow for formal releases to framework source code changes (but not necessarily for configuration changes).

It also adds a suggestion that we check with TNC prior to making a release to see whether there are any configuration changes they'd like to have incorporated in the release.

Connects #34

## Testing Instructions
- verify that the new release instructions are accurate
- verify that the wiki's "Configuration" page accurately describes the structure of the `config.json` file: https://github.com/CoastalResilienceNetwork/tnc-network-site/wiki/Configuration
- verify that the wiki's "Deployment" page accurately describes the steps TNC should take to change, commit, and deploy a new configuration: https://github.com/CoastalResilienceNetwork/tnc-network-site/wiki/Deployment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,11 @@ rules.
 
 ### Creating a Release with Git Flow
 
+This project uses git flow for releases of changes to site framework source
+changes. Prior to making a release, we should also ask TNC whether there are
```
Comment (caseycesari) [inline README.md]: `This project uses git flow for releases of changes to site framework source changes` reads a little off. Is the last `changes` supposed to `code`?

**Review by caseycesari — APPROVED** [review]
Documentation looks great! Nice job. My only suggestion is to add expected ranges (0 - 22?) for the zoom parameters.

File: README.md (hunk 1 above)
Comment (kellyi) [reply README.md]: Oops, yeah -- good catch. Will make a fix to update before merging.

**Comment by kellyi** [comment]
> My only suggestion is to add expected ranges (0 - 22?) for the zoom parameters.

Sounds good to me. I'll update the configuration doc with this.

**Comment by kellyi** [comment]
Noted the zoom range here https://github.com/CoastalResilienceNetwork/tnc-network-site/wiki/Configuration#initialmapzoom and here https://github.com/CoastalResilienceNetwork/tnc-network-site/wiki/Configuration#regionlist

Thanks for your help with this!

**Pull request merged**